### PR TITLE
feat: implement the e5 message

### PR DESCRIPTION
**Pull request by doudou**
[sc-40016]

### CODE REVIEW

File: lib/Core/Algorithm/src/EKF_Algorithm.c
Comment (wvmcastro) [inline lib/Core/Algorithm/src/EKF_Algorithm.c:1]: The filter's covariance matrix is the matrix `P` in `KalmanFilterStruct`; position, velocity and attitude covariances are block matrices on the main diagonal of `P`,
- position (3x3)
- velocity (3x3)
- attitude (4x4)

all of then are symmetric.

File: src/user/TidewiseMessages.c [hunk 1]
```diff
@@ -97,3 +97,70 @@ BOOL Fill_e4PacketPayload(uint8_t *payload, uint8_t *payloadLen) {
     return TRUE;
 }
 
+BOOL Fill_e5PacketPayload(uint8_t *payload, uint8_t *payloadLen) {
```
Comment (wvmcastro) [inline src/user/TidewiseMessages.c:100]: I think a call to this is missing

File: lib/Core/Algorithm/src/EKF_Algorithm.c
Comment (doudou) [reply lib/Core/Algorithm/src/EKF_Algorithm.c:1]: Fixed, thanks

File: src/user/TidewiseMessages.c (hunk 1 above)
Comment (doudou) [reply src/user/TidewiseMessages.c:100]: Also fixed.

**Review by wvmcastro — APPROVED** [review]

**Pull request merged**